integration_test/gce/gce_testing.go

```diff
@@ -1637,6 +1637,34 @@ func RestartInstance(ctx context.Context, logger *log.Logger, vm *VM) error {
 	return StartInstance(ctx, logger, vm)
 }
 
+// InstallGrpcurlIfNeeded installs grpcurl on instances that don't already have
+// it installed.
+func InstallGrpcurlIfNeeded(ctx context.Context, logger *log.Logger, vm *VM) error {
+	if IsWindows(vm.ImageSpec) {
+		return fmt.Errorf("installing grpcurl on Windows is not yet supported")
+	}
+
+	if _, err := RunRemotely(ctx, logger, vm, "which grpcurl"); err == nil {
+		return nil
+	}
+
+	logger.Printf("grpcurl not found, installing it...")
+
+	arch := "x86_64"
+	if IsARM(vm.ImageSpec) {
+		arch = "arm64"
+	}
+	if arch != "x86_64" && arch != "arm64" {
```

Comment:
Given how we define `arch` above, will we ever go into this if statement?

Reply:
You are right